pkg/extract/extract.go

```diff
@@ -181,7 +194,16 @@ func unzipFile(path string, file *zip.File) error {
 	}
 	defer targetFile.Close()
 
+	var reader io.Reader
+	reader = fileReader
+	if showProgress && file.FileInfo().Size() >= minSizeForProgressBar {
+		bar := pb.Simple.Start64(file.FileInfo().Size())
+		bar.Set("prefix", fmt.Sprintf("%s: ", filepath.Base(file.Name)))
+		defer bar.Finish()
+		reader = bar.NewProxyReader(fileReader)
+	}
```

Comment:
I believe this could go in a small helper shared with the tar code, especially as the `defer bar.Finish()` is not required, reader.Close() will also do it https://github.com/cheggaaa/pb/blob/ca7ab10606663e9efdbc3c1ad157459b5a2b0edf/reader.go#L21

Reply:
Good catch! I refactored a bit more and now I have https://github.com/code-ready/crc/pull/1343/files#diff-106940100fecd5f362aaeae069041830R198
The two functions are very similar but calling one from the other makes me introduce more complexity than required. WDYT ?